docs/src/pages/en/audio-player.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Audio player
+description: Build an audio player with Media Chrome
+layout: ../../layouts/MainLayout.astro
+---
+
+Specify the `audio` attribute on `<media-controller>` in order to use the default layout. When using the `audio` attribute the positioning slots like `"top-chrome"`, `"centered-chrome"`, etc. that are specified in the [Position controls](/en/position-controls) guide do not exist. There is only one default slot that all your markup can go into.
```

Comment:
The second sentence here feels a bit passive. A stab at reworking it:

"Using the `audio` attribute will remove the standard [positioning slots](/en/position-controls) from your player (eg: `"top-chrome"`, `"centered-chrome"`)."